test/unit/app/utils/ussd/kenya/test_user_profile_state.py

```diff
@@ -0,0 +1,97 @@
+import json
```

Comment:
These tests verify that the individual edits work - which I guess would be actions done _after_ registration is completed. But is that the same as testing the registration flow?

Reply:
The registration flow is tested in different scripts such as `test_misc_state.py` and `test_pin_state.py`